resource/airflow_dag_pipeline.yaml

```diff
@@ -49,6 +49,218 @@ jobs:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: amidos/dcind
+        inputs:
+          - name: image
+        run:
+          path: sh
+          args:
+            - -ec
+            - |
+              source /docker-lib.sh
+              start_docker
+              docker load -i image/image.tar
+              echo "Installing pytest"
+              docker run --entrypoint "" dag/test:latest pip install pytest
+              if [ -d "tests" ]; then
+                echo "Running tests in tests dir"
+                docker run --entrypoint "" dag/test:latest pytest tests
+              fi
+      on_failure:
+        put: pull-request
+        attempts: 3
+        params:
+          path: pull-request
+          status: failure
+          context: unittests
+    - task: run-syntax
+      privileged: true
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: amidos/dcind
+        inputs:
+          - name: image
+        run:
+          path: sh
+          args:
+            - -ec
+            - |
+              source /docker-lib.sh
+              start_docker
+              docker load -i image/image.tar
+              # Try importing files in base dir to check for syntax errors
+              docker run --entrypoint "" dag/test:latest python -c "
+              import os
+              from importlib import import_module
+              prefix = os.getcwd()
+              print(f'Working directory: {prefix}')
+              for dirpath, dirnames, filenames in os.walk(prefix):
+                  trimmed_mods = [f[:f.find('.py')] for f in filenames if not f.startswith('__') and f.find('.py') > 0]
+                  for m in trimmed_mods:
+                      mod = dirpath.replace(prefix + '/', '').replace('/', '.') + '.' + m
+                      print(f'importing: {mod}')
+                      import_module(mod)
+              "
+      on_failure:
+        put: pull-request
+        attempts: 3
+        params:
+          path: pull-request
+          status: failure
+          context: syntax
+    # mark test as passed
+    - put: pull-request
+      attempts: 3
+      params:
+        path: pull-request
+        status: success
+        context: iam-role
+    - put: pull-request
+      attempts: 3
+      params:
+        path: pull-request
+        status: success
+        context: unittests
+    - put: pull-request
+      attempts: 3
+      params:
+        path: pull-request
+        status: success
+        context: syntax
+  - aggregate:
+    - task: delete-iam-role-name
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: governmentpaas/awscli
+        inputs:
+          - name: pull-request-edit
+        params:
+          AWS_ACCESS_KEY_ID: ((secrets.role-putter-access-key-id))
```

Comment:
would prefer this to go into a dummy account

Reply:
or concat another policy that takes all permissions away

Reply:
I have added this to the Statements last in the poolicy

``` json
{
    "Sid": "denyall",
    "Action": [
        "*"
    ],
    "Effect": "Deny",
    "Resource": [
        "*"
    ]
}
```